packages/next-intl/prettier.config.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+/* eslint-disable import/no-extraneous-dependencies */
+
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'none',
+  bracketSpacing: false,
+  plugins: [require('prettier-plugin-tailwindcss')]
+};
```

Comment:
Sorry, there was some information missing here in the contributors guide.

I've just added [a new section about code formatting](https://github.com/amannn/next-intl/blob/main/CONTRIBUTORS.md#code-formatting-eslint--prettier).

Can you remove the two prettier configs you've added?